HospitalWebApp/Controllers/DoctorController.cs

```diff
@@ -30,7 +30,7 @@ public IActionResult GetBySpecialty(int specialtyId)
         }
 
         [HttpGet]
-        [Route("department/{id}")]
+        [Route("departments/{id}")]
```

Comment:
I think this should be department/id, like in method for specialty from above.

Reply:
it's all right, that's how it's defined in the [rules](https://restfulapi.net/resource-naming/) we'll use.